pkg/controller/nodes/task/taskexec_context_test.go

```diff
@@ -340,3 +351,70 @@ func TestConvertTaskResourcesToRequirements(t *testing.T) {
 		},
 	}, resourceRequirements)
 }
+
+func TestComputeRawOutputPrefix(t *testing.T) {
+
+	nCtx := &nodeMocks.NodeExecutionContext{}
+	nm := &nodeMocks.NodeExecutionMetadata{}
+	nm.OnGetOwnerID().Return(types.NamespacedName{Namespace: "namespace", Name: "name"})
+	nCtx.OnOutputShardSelector().Return(ioutils.NewConstantShardSelector([]string{"x"}))
+	nCtx.OnRawOutputPrefix().Return("s3://sandbox/")
+	ds, err := storage.NewDataStore(
+		&storage.Config{
+			Type: storage.TypeMemory,
+		},
+		promutils.NewTestScope(),
+	)
+	assert.NoError(t, err)
+	nCtx.OnDataStore().Return(ds)
+	nCtx.OnNodeExecutionMetadata().Return(nm)
+
+	pre, uid, err := ComputeRawOutputPrefix(context.TODO(), 100, nCtx, "n1", 0)
+	assert.NoError(t, err)
+	assert.Equal(t, uid, "name-n1-0")
```

Comment:
```suggestion
	assert.Equal(t, "name-n1-0", uid)
```
change order of args because test output uses first for "expected" and second for "observed"

Reply:
Yup